src/renderer/base/renderer.cpp

```diff
@@ -900,6 +911,22 @@ void Renderer::_PaintBufferOutputGridLineHelper(_In_ IRenderEngine* const pEngin
 {
     // Convert console grid line representations into rendering engine enum representations.
     IRenderEngine::GridLines lines = Renderer::s_GetGridlines(textAttribute);
+
+    // For now, we dash underline patterns and switch to regular underline on hover
+    if (_pData->GetPatternId(coordTarget).size() > 0)
+    {
+        if (_hoveredInterval.has_value() &&
+            _hoveredInterval.value().start <= til::point{ coordTarget } &&
+            til::point{ coordTarget } <= _hoveredInterval.value().stop)
+        {
+            lines |= IRenderEngine::GridLines::Underline;

```

Comment:
WI_SetFlag here and below?

Reply:
We use `|=` in `s_GetGridlines`, I'd like to keep it this way for consistency

Reply:
Sure. fine.